creator-cards/mohammed-alwedaei.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Mohammed Alwedaei",
+  "type": "CREATOR",
+  "tags": ".NET, C#, ASP.NET, Cloud, Architecture",
+  "github": "https://github.com/Mohammed-Alwedaei",
+  "channels": [
+    {
+      "name": "Site",
+      "url": "https://www.pexelx.io/"
+    },
+    {
+      "name": "Youtube",
+      "url": "https://www.youtube.com/@developwithpixelx"
+    }
+  ],
+  "feeds": [
+    {
+      "type": "LinkTagRssType",
```

Comment:
choose dotnet here as type. Since type = LinkTagRssType is fom the old card format.

Reply:
Ok gonna change it